ci-operator/step-registry/ipi/conf/azurestack/ipi-conf-azurestack-ref.yaml

```diff
@@ -25,12 +25,17 @@ ref:
       These sizes are roughly consistent across all cloud providers, but we may not be able to instantiate some sizes
       in some regions or accounts due to quota issues.
   - name: AZURESTACK_ENDPOINT
-    default: 'https://management.ppe3.stackpoc.com'
+    default: 'https://management.mtcazs.wwtatc.com'
     documentation: |-
       The Azure Stack Hub Endpoint to connect to.
   - name: SUFFIX_ENDPOINT
-    default: "ppe3.stackpoc.com"
+    default: "mtcazs.wwtatc.com"
     documentation: |-
       The endpoint suffix for storage accounts.
+  - name: CLUSTER_OS_IMAGE
```

Comment:
this parameter seems not used in step `ipi-conf-azurestack`, I guess we don't need to define it here.